src/RelayClient.ts

```diff
@@ -150,12 +155,14 @@ class RelayClient extends EnvelopingEventEmitter {
       throw new Error('No call verifier present. Check your configuration.');
     }
 
-    const relayServerUrl = this._envelopingConfig.preferredRelays.at(0);
+    if (!this._relayServerUrl) {
+      const relayServerUrl = this._envelopingConfig.preferredRelays.at(0);
 
-    if (!relayServerUrl) {
-      throw new Error(
-        'Check that your configuration contains at least one preferred relay with url.'
-      );
+      if (!relayServerUrl) {
```

Comment:
```suggestion
      if (!this._envelopingConfig.preferredRelays.length) {
```
And will remove the variable.

Reply:
Changed.